base/iterators.jl

```diff
@@ -1476,4 +1476,97 @@ only(x::NamedTuple) = throw(
     ArgumentError("NamedTuple contains $(length(x)) elements, must contain exactly 1 element")
 )
 
+"""
+    unfold(f, initialstate, [eltype])
+
+Iterable object that generates values from an initial state and a transition
+function `f(state)`. The function must follow the same rules as `iterate`.
+It returns either `(newvalue, newstate)` or `nothing`, in which case the
+sequence ends.
+
+The optional parameter `eltype` can specify the type of the generated values.
+
+See also: [`iterate`](@ref)
```

Comment:
```suggestion
See also: [`iterate`](@ref), [the iteration interface](@ref man-interface-iteration)
```

I'm not sure I've got the syntax right there for a link to a manual section, but you get the idea.

Reply:
I'm not sure of the syntax either.

Reply:
Seems to match this at least https://github.com/JuliaLang/julia/blob/7ef78e45b3aa7fb523919fe6c6dd3182b88d5b6c/doc/src/manual/functions.md?plain=1#L412